server/app/services/program/StatusDefinitions.java

```diff
@@ -0,0 +1,71 @@
+package services.program;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.google.auto.value.AutoValue;
+import com.google.common.collect.ImmutableList;
+import java.util.Optional;
+import services.LocalizedStrings;
+
+public class StatusDefinitions {
```

Comment:
Please add a JavaDoc describing the use and purpose.

Reply:
done